st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -435,7 +435,6 @@ def _sanitize_trigger(self, trigger):
     if not isinstance(parent_args, list):
         raise TypeError(
             "Command line arguments passed to the parent process must be a list"
-            f" (was {type(parent_args)})."
```

Comment:
Why does this cause a syntax error?

Reply:
Also in the original PR change, it was requested that as part of the error it reports the type of the argument that isn't a list, so that it enables easier debugging. So even if it needs changing we still need that type in there.

It's odd - as I can run that block of code lines 435-438 perfectly fine.

Are we definitely sure its being run in a python 3.6 or python 3.8 environment?